shimmingtoolbox/cli/mask.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python3
+
+import click
+import nibabel as nib
+import numpy as np
+import os
+
+import shimmingtoolbox.masking.threshold
+from shimmingtoolbox.masking.shapes import shape_square
+from shimmingtoolbox.masking.shapes import shape_cube
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.group(context_settings=CONTEXT_SETTINGS, help=f"Creates a cube, square or threshold mask.")
+def mask():
+    pass
+
+
+@mask.command(context_settings=CONTEXT_SETTINGS,
+              help=f"Creates a cube mask from the input file. Returns an output nifti file with cube mask.")
+@click.option('-input', 'fname_input', type=click.Path(), required=True, help="Complete input path of the nifti file to"
+                                                                              " mask")
+@click.option('-output', type=click.Path(), default=os.curdir, help="Output folder for mask in nifti file")
+@click.option("-len_dim1", required=True, type=int, help="Length of the side of the cube along first dimension (in "
+                                                         "pixels)")
+@click.option("-len_dim2", required=True, type=int, help="Length of the side of the cube along second dimension (in "
+                                                         "pixels)")
+@click.option("-len_dim3", required=True, type=int, help="Length of the side of the cube along third dimension (in "
+                                                         "pixels)")
+@click.option("-center_dim1", type=int, default=None, help="Center of the cube along first dimension (in "
+                                                           "pixels). If no center is provided, the middle is "
+                                                           "used.")
+@click.option("-center_dim2", type=int, default=None, help="Center of the cube along second dimension (in "
+                                                           "pixels). If no center is provided, the middle is "
+                                                           "used.")
+@click.option("-center_dim3", type=int, default=None, help="Center of the cube along third dimension (in "
+                                                           "pixels). If no center is provided, the middle is "
+                                                           "used.")
```

Comment:
it's heavy to have to enter those dims separately. you could have a simpler approach:
~~~
st_mask -center x y z
~~~

Reply:
I adapted it this way, thank you!